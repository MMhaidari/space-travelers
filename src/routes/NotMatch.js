import React from 'react';

const NotMatch = () => (
  <div>NOT Found</div>
);

export default NotMatch;
