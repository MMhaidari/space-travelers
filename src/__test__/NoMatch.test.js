import React from 'react';
import renderer from 'react-test-renderer';
import NotMatch from '../routes/NotMatch';

describe('NotMatch', () => {
  it('should render the "Not Found" text', () => {
    const component = renderer.create(<NotMatch />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
