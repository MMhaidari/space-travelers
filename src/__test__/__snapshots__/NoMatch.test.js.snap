// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotMatch should render the "Not Found" text 1`] = `
<div>
  NOT Found
</div>
`;
